core/src/main/scala/io/scalac/mesmer/core/akka/model/AttributeNames.scala

```diff
@@ -0,0 +1,8 @@
+package io.scalac.mesmer.core.akka.model
+
+object AttributeNames {
```

Comment:
If we follow this rule rigorously we'll end up with a single module and this will get tricky to work with. I imagine that in the future we will get modules for all every technology that we support with core module shared by all of those - this is something that might be shared for all instrumentations. I don't really like the idea of putting everything in the single module TBH

Reply:
Note we can have clean separation not only on the module level but also on the package level.

Right now: 
 - there's no need for modules other than trying to be future-proof and the future requiring such a split will most probably never come
 - we have code split between modules so we don't really see at the first glance how much code is required to actually instrument something.
 - there's code that gets forgotten and unused (eg. JarNames object). Since we look at core less often, there's a smaller chance it will be removed/maintained etc
 - we have an additional build.sbt code to read. sbt code is hard to follow so if possible it should be as simple as possible (that's minor but still valid)

> we'll end up with a single module and this will get tricky to work with. 

why would that be tricky given a clean separation between packages?

I think a sensible common ground would be just to keep core at the necessary minimum and only keep things like typeclasses ("Traverse", "Combine", "encode"), utils (time related, randomness related etc), whereas akka related/zio related stuff should land in the extension directly. wdyt? So I'd still suggest AtttributeNames migration.